esprima.js

```diff
@@ -1274,27 +1281,13 @@
 
                     default:
                         if (isOctalDigit(ch)) {
```

Comment:
It’s just some background on the history of octals/noctals. For a long time, e.g. `\08` in a string literal was supposed to throw an error (per the spec), but engines in the wild treated it as `\x008`.

Let’s hope implementors won’t make the same mistake when it comes to template literals. https://bugs.ecmascript.org/show_bug.cgi?id=1553 is a better link, actually (removed the old one to remove confusion).